sahi/model.py

```diff
@@ -1065,3 +1064,104 @@ def _create_object_prediction_list_from_original_predictions(
             object_prediction_list_per_image.append(object_prediction_list)
 
         self._object_prediction_list_per_image = object_prediction_list_per_image
+
+
+@check_requirements(["tensorflow", "tensorflow_hub"])
+class TensorflowHubDetectionModel(DetectionModel):
+    def load_model(self):
+        import tensorflow_hub as hub
+
+        if "tfhub.dev/tensorflow" in self.model_path and "object_detection" in self.model_path:
```

Comment:
burayi yanlis dusunmusum

```suggestion
        if "tfhub.dev/tensorflow" in self.model_path:
```